src/Symfony/Component/Form/Tests/AbstractBootstrap3HorizontalLayoutTest.php

```diff
@@ -100,6 +100,16 @@ public function testLabelWithCustomTextAsOptionAndCustomAttributesPassedDirectly
         );
     }
 
+    public function testLabelNoDivIfFalse()
+    {
+        $this->assertEmpty(
```

Comment:
I second that. See [more information about thoughtless one-time variables here](https://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html). 

If you need the variable because it clarifies things, maybe consider using a comment instead, you can even write whole sentences with them.